assets/dev/js/frontend/handlers/base-tabs.js

```diff
@@ -163,6 +163,7 @@ export default class baseTabs extends elementorModules.frontend.handlers.Base {
 						break;
 					case 'Enter':
 					case 'Space':
+					case ' ': // Space.
```

Comment:
Why is it here?

Reply:
I don't know what does the `'Space'` below but when i debugged it, the event is `' '`.